web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -330,20 +332,206 @@ Once we confirm everything works well locally, we will apply the same changes to
 
 You can follow these steps to migrate to the new auth system (assuming you already migrated the project structure to 0.12, as described [above](#migrating-your-project-to-the-new-structure)):
 
+1. If you used `getUserFieldsFn` and/or `additionalSignupFields`, you will need to migrate them to the new `userSignupFields` field.
+
+  In Wasp 0.11.X, you could define a `getUserFieldsFn` to specify extra fields that would get saved to the `User` when using Google or GitHub to sign up.
+    
+  You could also define `additionalSignupFields` to specify extra fields for the Email or Username & Password signup process.
+
+  In 0.12.X, we unified these two concepts into the `userSignupFields` field.
+
+  <details>
+  <summary>Migration for <EmailPill /> and <UsernameAndPasswordPill /></summary>
+
+    First, move the value of `auth.signup.additionalFields` to `auth.methods.{method}.userSignupFields` in the `main.wasp` file.
+      
+    `{method}` depends on the auth method you are using. For example, if you are using the email auth method, you should move the `auth.signup.additionalFields` to `auth.methods.email.userSignupFields`.
+    
+    To finish, update the implementation to use the `defineUserSignupFields` from `wasp/server/auth` instead of `defineAdditionalSignupFields` from `@wasp/auth/index.js` (this should have been automatically done by the migration script).
```

Comment:
```suggestion
    To finish, update the JS/TS implementation to use the `defineUserSignupFields` from `wasp/server/auth` instead of `defineAdditionalSignupFields` from `@wasp/auth/index.js`.
```

It is not done by the script currently, and maybe it is best for them to do it manually, just to be sure all is making sense, so I just removed this part about the migration script.